### ISSUE TITLE
Removing a concept from concept set in existing cohort upon removal lands on configuration page

### ISSUE THREAD

**Issue by jhardin29** [OHDSI/Atlas#887]
### Expected behavior
 Open concept set in existing cohort, remove a concept, page should remain on the concept set page updated to illustrate removal of the given individual concept.
### Actual behavior
Open concept set in existing cohort, remove a concept, page refreshes and lands on ATLAS configuration page. 
### Steps to reproduce behavior
Open concept set in existing cohort, remove a concept.

**Comment by chrisknoll** [comment]
This is a critical issue.  From the console, it is an error calling 'resolve concept set expression'.

```
Can not construct instance of org.ohdsi.circe.vocabulary.ConceptSetExpression: no String-argument constructor/factory method to deserialize from String value ('{"items":[{"concept":{"CONCEPT_ID":21500520,"CONCEPT_NAME":"Prostatic Hypertrophy Agent - 5-alpha Reductase Inhibitors","STANDARD_CONCEPT":"C","STANDARD_CONCEPT_CAPTION":"Classification","INVALID_REASON":"V","INVALID_REASON_CAPTION":"Valid","CONCEPT_CODE":"00000520","DOMAIN_ID":"Drug","VOCABULARY_ID":"ETC","CONCEPT_CLASS_ID":"ETC"},"isExcluded":false,"includeDescendants":true,"includeMapped":false}]}')
 at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@3bc9d3ae; line: 1, column: 1]
```

I think this is due to sending the content type of 'string' instead of JSON.

**Comment by chrisknoll** [comment]
Fixed in  57a43e6.
